### ISSUE TITLE
Tip of the day - Learn more says user manual is missing

### ISSUE THREAD

**Issue by hellsworth** [glimpse-editor/Glimpse#170]
Clicking the Learn more link on the first tip doesn't do anything at all and on all of the following tips, it pops up a new window saying the user manual is missing.

![manual-missing](https://user-images.githubusercontent.com/12970280/66167600-c2d60700-e5f7-11e9-9967-d03034a96f06.png)

**Comment by hellsworth** [comment]
actually this may not be a problem to point the user to read the documentation online. However, when I click "Read Online", I get a pop up message.

![read-online-error](https://user-images.githubusercontent.com/12970280/66167738-4132a900-e5f8-11e9-9c4b-4e211adc0870.png)

**Comment by TrechNex** [comment]
cc @bochecha as this looks potentially like a flatpak-related issue

**Comment by hellsworth** [comment]
ah actually, I'm running the snap. Let me build this in flatpak real quick to test that too.

**Comment by TrechNex** [comment]
Ah okay! In that case also linking to this PR I see you commented on earlier! #153

**Comment by TrechNex** [comment]
For background the offline help pages are hosted by a plug-in. When they're pointed to the "gimp" browser (line 774 in `etc/gimprc.in` defines the default value for the file hosted in `~/.config/glimpse/`) they rely on a built version of webkit for GTK2 being present.

Some of the work @bochecha has done over the last couple of days is changing the way Flatpaks are packaged so we can avoid needing that dependency in the development builds.

**Comment by bochecha** [comment]
Right, so not having WebKit is responsible for the first popup. Since without WebKit the help browser plugin is not built, you get the "The Glimpse user manual is not installed" popup.

However, clicking on the "Read Online" button is supposed to take you to the online help in your favourite web browser, whether the build has WebKit or not.

So that second failure popup means your build doesn't manage to open a page in your web browser. I tested that this works in Flatpak, so this seems like a Snap-specific issue.

**Comment by hellsworth** [comment]
> so this seems like a Snap-specific issue.

This is excellent to know. Shall we keep this issue open and label it with snap? Or close it since snap support isn't yet merged? If it gets closed, @psymole and myself will just have to make sure a web browser can be opened from the snap. I vote for the first option, just so it doesn't get forgotten about :)

**Comment by Member1221** [comment]
giving it the packaging tag, since it's a snap issue. :)

**Comment by TrechNex** [comment]
Is this bug something that still needs to be fixed, or can we close it?

**Comment by TrechNex** [comment]
Moved as a 0.2.0 candidate as not necessary for 0.1.0 to release
